src/analysis/retail/paladin/holy/modules/beacons/BeaconHealSource.ts

```diff
@@ -199,6 +212,54 @@ class BeaconHealSource extends Analyzer {
       });
     });
   }
+
+  _guessUntracked(beaconTransferEvent: HealEvent) {
+    const beaconTransferRaw =
+      beaconTransferEvent.amount +
+      (beaconTransferEvent.absorbed || 0) +
+      (beaconTransferEvent.overheal || 0);
+
+    const fmtAbility = (ability: HealEvent['ability']) => ({
+      name: ability.name,
+      id: ability.guid,
+      icon: ability.abilityIcon.replace('.jpg', ''),
+    });
+
+    if (this.unconfiguredHeals.length) {
+      const match = this.unconfiguredHeals.findLast((healEvent) => {
+        const rawHealing = healEvent.amount + (healEvent.absorbed || 0) + (healEvent.overheal || 0);
+        const expectedBeaconTransfer = rawHealing * this.beaconTransferFactor.beaconFactor;
+
+        return (
+          Math.abs(beaconTransferEvent.timestamp - healEvent.timestamp) < 500 &&
+          Math.abs(expectedBeaconTransfer - beaconTransferRaw) <= 2 // allow for rounding errors on Blizzard's end
+        );
+      });
+
+      if (match) {
+        this.log('Likely match:', { ability: fmtAbility(match.ability), healEvent: match });
+        return;
+      }
+
+      const matchHalf = this.unconfiguredHeals.findLast((healEvent) => {
```

Comment:
since you're only looking at unconfigured heals this list is probably not going to be too long but noting that this could have pretty bad performance if that ever changes